app/src/main/java/com/plivo/voicecalling/Helpers/Phone.java

```diff
@@ -33,11 +33,12 @@ public static Phone getInstance(EndPointListner endPointListener) {
     private Phone() {
 
         endpoint = new Endpoint(true, this);
+        endpoint.setRegTimeout(479);
```

Comment:
add comment / description for 8 min timeout

Reply:
done